src/Analysis/GWASV2/Components/DismissibleMessage/DismissibleMessage.jsx

```diff
@@ -19,11 +24,9 @@ const DismissibleMessage = ({
             tabIndex='0'
             role='button'
             aria-label='Close Message'
-            onClick={() => {
-              setOpen(false);
-            }}
+            onClick={close}
             onKeyDown={(e) => {
-              if (isEnterOrSpace(e)) setOpen(false);
+              if (isEnterOrSpace(e)) close();
```

Comment:
If the user closes the message manually, do we also want to remove it from the messages list in the global state? Its not immediately clear to me if that is occurring in the close() function all of the time since there's a conditional there.

Reply:
if it is not removed from the global state, it will stay on the screen, since we have this in our app:
```js
      <DismissibleMessagesList
        messages={state.messages}
        dismissMessage={(chosenMessage) => {
          dispatch({
            type: ACTIONS.DELETE_MESSAGE,
            payload: chosenMessage,
          });
        }}
      />
```
i.e. the messages are always rendered based on `state.messages`